README.md

```diff
@@ -1,16 +1,66 @@
-<p align="center"><img src="figures/Cesium3DTiles.png" /></p>
+<p align="center"><img src="./figures/Cesium3DTiles.png" /></p>
 
-Specification for streaming massive heterogeneous **3D** geospatial datasets.
+An open specification for streaming massive heterogeneous 3D geospatial datasets.
 
 3D Tiles has entered the Open Geospatial Consortium (OGC) [Community Standard](https://cesium.com/blog/2016/09/06/3d-tiles-and-the-ogc/) process.
 
----
+## Overview
 
-Created by the <a href="http://cesiumjs.org/">Cesium team</a> and built on <a href="https://www.khronos.org/gltf">glTF</a>.<br/>
+Bringing techniques from graphics research, the movie industry, and the game industry to 3D geospatial, 3D Tiles define a spatial data structure and a set of tile formats designed for 3D and optimized for streaming and rendering. 
```

Comment:
Since this is the new landing page for 3D Tiles,  I figured there should be some sort of introductory section. I'l edit the tone and add a sample of domains.